src/view/WorkspaceManager.js

```diff
@@ -191,6 +201,7 @@ define(function (require, exports, module) {
      */
     Panel.prototype.hide = function () {
         Resizer.hide(this.$panel[0]);
+        exports.trigger(EVENT_WORKSPACE_PANEL_HIDDEN, this.panelID);
```

Comment:
I am not sure if having a new event is easier to deal with. With the param passing approach we have one event less. I don't see a compelling reason why we would want two events when one can suffice the requirement.